Command/RemoveThumbsCommand.php

```diff
@@ -127,12 +123,9 @@ public function execute(InputInterface $input, OutputInterface $output)
                     continue;
                 }
                 //clean filesystem registry for saving memory
-                $fsRegister->setValue($filesystem, []);
+                $provider->getFilesystem()->clearFileRegister();
             }
 
-            //clear entity manager for saving memory
-            $this->getMediaManager()->getEntityManager()->clear();
```

Comment:
Why did you delete this line? I don't know if it effectivly saves memory or if it is safe. Is it the reason?

Maybe it won't work or it is useless if some media managers have a different entity manager?

Reply:
As of `"sonata-project/core-bundle": "^3.2"`, `Sonata\CoreBundle\Model\ManagerInterface` (which is returned by `$this->getMediaManager()` does not contain `getEntityManager` method.

Reply:
We should keep this line. Maybe add a `cleanup` method to the `MediaManager` to do the same.